CHANGELOG.md

```diff
@@ -3,6 +3,120 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.0
+
+_Mar 3, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
```

Comment:
Let's make this a complete sentence.

```suggestion
We'd like to offer a big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
```

Reply:
We always write the same introduction. I'm adding the modification to the changelog generator such that future message will get the modification too